packages/e2e/src/specs/contacts/contacts-in-app-navigation.e2e.ts

```diff
@@ -52,22 +52,22 @@ describe("Contacts screen check", () => {
   })
 
   it("Should check if 'phone number' input field is displayed", async () => {
-    const phoneNumber = await ContactsPage.phoneNumberInput
+    const phoneNumber = await ContactsPage.primaryPhoneNumberInput
```

Comment:
🟢  please rename `phoneNumber` to `primaryPhoneNumberInput` and do similar changes for constants in following tests

Reply:
added for refactor task-> CP-1889